uui-components/src/inputs/NumericInput.tsx

```diff
@@ -60,6 +60,12 @@ const getFractionDigits = (formatOptions: Intl.NumberFormatOptions) => {
     return maximumFractionDigits;
 };
 
+const toFixedWithoutRoundingUp = (value: number, fractionDigits: number) => {
```

Comment:
Let's move this function into `uui-core/src/helpers/numericInputCalculations.ts `and also add tests for it